12/umbraco-cms/tutorials/starter-kit/lessons/3-add-open-graph/step-4.md

```diff
@@ -73,7 +73,7 @@ The final view should look like this:
 {% hint style="info" %}
 If you do not have Starter Kit installed, or experience issues with the properties not being picked up/showing up in HTML feel free to remove the fallback to siteName.
 
-If your meta properties do not show up on social media, make sure to inspect source HMTL and make sure there are no inline HTML tags in `og:title`, `og:description` etc.
+If your meta properties do not show up on social media, make sure to inspect source HTML. Make sure there are no inline HTML tags in `og:title`, `og:description` etc.
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?